tests/test_summarize_categorical_features.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
Description for top of script:
```suggestion
"""Unit testing for the summarize_categorical_features.py script"""
import pytest
```